python/sglang/srt/mem_cache/memory_pool.py

```diff
@@ -42,19 +42,14 @@
 import triton.language as tl
 
 from sglang.srt.constants import GPU_MEMORY_TYPE_KV_CACHE
+from sglang.srt.disaggregation.mooncake.utils import init_mooncake_custom_mem_pool
```

Comment:
```suggestion
from sglang.srt.utils import maybe_init_custom_mem_pool
```

Then we define a function in sglang.srt.utils:
```python
def maybe_init_custom_mem_pool(
    device: str,
) -> Tuple[bool, Optional[Any], Optional[str]]:
    # This function can be modified to support more features that require a custom memory pool.
    enable_custom_mem_pool = True if envs.SGLANG_MOONCAKE_CUSTOM_MEM_POOL.get() is not None else False

    if enable_custom_mem_pool:
        # Currently, only mooncake requires a custom mem pool for MNNVL PD disaggregation
        from sglang.srt.disaggregation.mooncake.utils import init_mooncake_custom_mem_pool
        return init_mooncake_custom_mem_pool(device)
    else:
        return False, None, None
```

Reply:
Just this minor suggestion, others LGTM.